tests/databases/constructed_molecule/fixtures/constructed_molecule_mongo_db.py

```diff
@@ -98,6 +98,28 @@
                         ),
                     ),
                 ),
+                lru_cache_size=0,
+            ),
+            molecule=stk.ConstructedMolecule(
+                topology_graph=stk.polymer.Linear(
+                    building_blocks=(
+                        stk.BuildingBlock(
+                            smiles='Br[C+2][C+2]Br',
+                            functional_groups=[stk.BromoFactory()],
+                        ),
+                    ),
+                    repeating_unit='A',
+                    num_repeating_units=2,
+                ),
+            ),
+            key={'SMILES': 'Br[C+2][C+2][C+2][C+2]Br'},
+        ),
+        CaseData(
+            database=stk.ConstructedMoleculeMongoDb(
+                mongo_client=MockMongoClient(),
+                jsonizer=stk.ConstructedMoleculeJsonizer(
+                    key_makers=(stk.Smiles(),)
+                ),
```

Comment:
I don't think we need a new test case for a constructed molecule mongo db - since you didnt change that class.